### PR TITLE
Fix zdoom polyobj rendering crash in software 

### PR DESCRIPTION
**Pull request by bkoropoff**

### CODE REVIEW

File: prboom2/src/hexen/po_man.c [hunk 1]
```diff
@@ -66,7 +66,7 @@ static void ResetSegDrawingParameters(seg_t *seg)
   seg->v1->py = seg->v1->y;
   seg->v2->px = seg->v2->x;
   seg->v2->py = seg->v2->y;
-  seg->pangle = seg->angle;
+  seg->pangle = R_PointToAngle2(seg->v1->px, seg->v1->py, seg->v2->px, seg->v2->py);
```
Comment (kraflab) [inline prboom2/src/hexen/po_man.c:69]: This definitely only affects rendering right?

Comment (bkoropoff) [reply prboom2/src/hexen/po_man.c:69]: Yeah, it gets used in one place in r_segs.c for rendering

**Comment by kraflab** [comment]
👍

**Comment by kraflab** [comment]
Oops, this does break hexen 🙂

**Pull request merged**